src/test/java/com/snack/news/service/AdminServiceTest.java

```diff
@@ -17,9 +17,9 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.mockito.ArgumentMatchers'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4554642)